src/BinDeps.jl

```diff
@@ -1,3 +1,4 @@
+VERSION >= v"0.4.0" && __precompile__()
```

Comment:
`isdefined(Base, :__precompile__)` to be more specific and also address the problem for later prereleases?